using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -348,7 +348,7 @@ ACPAnalytics.setVisitorIdentifier("yourVisitorId");
 Before using this API, see [Identify unique visitors](https://docs.adobe.com/content/help/en/analytics/implementation/javascript-implementation/unique-visitors/visid-overview.html).
 {% endhint %}
 
-Sets a custom Analytics visitor identifier. For more information, see [Custom Visitor ID](https://marketing.adobe.com/resources/help/en_US/sc/implement/visid_custom.html).
+Gets a custom Analytics visitor identifier which has been set previously using [setVisitorIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#setidentifier).
```

Comment:
- Please add "ThIs API" at the beginning of the 1st sentence...(part of the fixing the gaps project).
- Please add a comma between "identifier" and "which